tests/README.md

```diff
@@ -54,6 +54,14 @@ Runtime tests will call the bpftrace executable. These are located in `tests/run
 Runtime tests are grouped into "suites". A suite is usually a single file. The
 name of the file is the name of the suite.
 
+### Comments
```

Comment:
Put this after "Runtime test directives" section? Better to order this from most to least important